resources/assets/sass/components/_nav.scss

```diff
@@ -0,0 +1,89 @@
+.c-nav {
+}
+
+	.c-nav__list {
+
+		@include clearfix;
+		@include clear-list;
+	}
+
+		.c-nav__list-item {
+
+			float: left;
+			display: block;
+			margin: 12px;
+			height: 36px;
+
+			&.m-has-children {
+
+				@include arrowY(down, $color-white);
```

Comment:
Name of mixin `arrowY` should be written in all lowercase letters with hyphens instead of underscores